runtime/src/rpc.rs

```diff
@@ -169,7 +165,7 @@ impl PolkaBtcProvider {
     {
         let mut sub = self.ext_client.subscribe_finalized_blocks().await?;
         loop {
-            on_block(sub.next().await).await?;
+            on_block(sub.next().await.unwrap()).await?;
```

Comment:
Is this safe?

Reply:
| Returns the next notification from the stream This may return None if the subscription has been terminated, may happen if the channel becomes full or is dropped.